app.py

```diff
@@ -0,0 +1,29 @@
+import datetime
+import json
+
+import requests
+from flask import Flask
+
+app = Flask(__name__)
+
+
+@app.route('/sun/')
+def sun_info():
+    header = {"x-access-token": "9a1770975b443e98b4f04a27eead7eb8"}
+    day = datetime.date.today()
+    url = 'https://api.openuv.io/api/v1/uv?lat=56.33&lng=44.01&dt=' + str(day)
```

Comment:
What coordinates did you use here? Maybe better, use variables for setting position